src/image.js

```diff
@@ -0,0 +1,121 @@
+require("babel-polyfill");
+
+import gm from "gm";
+import log from "./log";
+import uuid from "uuid";
+
+const im = gm.subClass({imageMagick: true});
+
+// Wrap these calls in promises so we can use async/await
+const promiseWrite = (client, file) => {
+  return new Promise((resolve, reject) => client.write(file, err => err ? reject(err) : resolve()));
+};
+const promiseSize = (client) => {
+  return new Promise((resolve, reject) => client.size((err, data) => err ? reject(err) : resolve(data)));
+};
```

Comment:
Same comment as I placed above: the `promise` in the function names doesn't add anything and, IMHO, only makes it more confusing.